core/commands/check_relevance_command.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'base_command'
+
+# This class checks relevance network config
+class CheckRelevanceCommand < BaseCommand
+  def self.synopsis
```

Comment:
This synopsis should be used in the `HelpCommand`. You should also get the string of the synopsis from the HelpCommand itself, it is better written there.